java/arcs/sdk/android/dev/api/CollectionProxy.java

```diff
@@ -285,6 +285,11 @@ public String getId() {
     public Referencable tryDereference() {
       return this;
     }
+
+    @Override
+    public void setExpiration(long expirationTimestamp) {
+      throw new AssertionError("ModelEntry::setExpiration not implemented");
```

Comment:
You can just use `TODO("ModelEntry::setExpiration not implemented")` instead of throwing an exception.

Reply:
this is a java class, i can't have `TODO("...")` in java :(

Reply:
oh. right... oops. You could convert it to kotlin? 🙃

Reply:
i prefer to delete it altogether 😺